prototypes/item.lua

```diff
@@ -111,20 +111,6 @@ data:extend({
         straight_rail = "straight-water-way",
         curved_rail = "curved-water-way"
         },
-        {

```

Comment:
I'm not quite sure why but this seems to be the cause of the pipette not working on correctly on straight water ways. I don't see any new bugs introduced by removing it after play for about 30 minutes and working on my ship routes. What was the reason for adding it?

Reply:
to be honest i was moving into pretty unfimilar waters ( :D ) when introducing the waterways, and copied aproaches from some other mod when implementing the waterways, including the addition of the new railplaner. 
so if it turns out to be not neccesarry and even causing bugs it can definitly be removed.
i will test and probably merge your changes this weekend!